src/utils/formUtils.js

```diff
@@ -97,7 +97,7 @@ export function eachComponent(components, fn, includeAll, path, parent) {
     if (!noRecurse) {
       if (hasColumns) {
         component.columns.forEach((column) =>
-          eachComponent(column.components, fn, includeAll, subPath(), parent ? component : null));
```

Comment:
This just tells me that the component JSON is incorrect. Columns should be an array of objects that contain their own "components" property.

Reply:
@travist , the compoent's JSON is correct. It is an array of objects that containt  their own "components" property. But the **instance** of the Columns component has it own 'columns' property where each column is jsut an array of components' instances within the column.